lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.spec.ts

```diff
@@ -210,11 +210,11 @@ describe('TaskHeaderCloudComponent', () => {
             expect(taskCloudService.updateTask).toHaveBeenCalled();
         }));
 
-        it('should show loading spinner when properties are not loaded', () => {
-            component.properties = null;
+        it('should show spinner before loading task details', () => {
+            component.isLoading = true;
             fixture.detectChanges();
             const loading = fixture.debugElement.query(By.css('.adf-task-header-loading'));
-            expect(loading).toBeTruthy();
+            expect(loading).not.toBe(null);
```

Comment:
this can still be undefined maybe is better check to be toBeTruthy

Reply:
Updated